lnbits/core/views/api.py

```diff
@@ -697,3 +699,34 @@ async def api_auditor(wallet: WalletTypeInfo = Depends(get_key_type)):
         "delta_msats": delta,
         "timestamp": int(time.time()),
     }
+
+
+##################UNIVERSAL WEBSOCKET MANAGER########################
+
+
+@core_app.websocket("/api/v1/ws/{item_id}")
+async def websocket_connect(websocket: WebSocket, item_id: str):
+    await websocketManager.connect(websocket, item_id)
+    try:
+        while True:
+            data = await websocket.receive_text()
+    except WebSocketDisconnect:
+        websocketManager.disconnect(websocket)
+
+
+@core_app.post("/api/v1/ws/{item_id}")
+async def websocket_update_post(item_id: str, data: str):
```

Comment:
- should it allow multiple path parameters (like `/api/v1/ws/aaa/bbb/ccc`)?
- if yes then maybe [:path](https://www.starlette.io/routing/#path-parameters) should be used:

> path returns the rest of the path, including any additional / characters.
```
Route('/uploaded/{rest_of_path:path}', uploaded)
```

Reply:
No, its not needed